deploy/roles/r-libs/defaults/main.yml

```diff
@@ -0,0 +1,34 @@
+---
+install_r_updates: true
+install_r_packages: true
+
+r_dependencies:
+  - QuantLib
+  - QuantLib-devel
+  - libcurl-devel
+  - libxml2-devel
+  - hunspell-devel
+  - udunits2
+  - udunits2-devel
+
+r_packages:
+  - "devtools"
+#  - "http://cran.r-project.org/src/contrib/Archive/RQuantLib/RQuantLib_0.3.12.tar.gz" # The latest version requires QuantLib 1.4.0, which is not supported by Centos 6.5
```

Comment:
@lel99999 Let's also remove this line and the QuantLib dependencies above, since it has issues and we can use "devtools" for Vagrant testing anyways.